### DISCUSSION TITLE
[Script request]  Auto-mount drive

### DISCUSSION THREAD

**Discussion by thor9898** [community-scripts/ProxmoxVE#998] [Request script]
### Application Name

/etc/fstab

### Website

--

### Description

First, I want to express my gratitude to everyone who has contributed to this project, especially the late tteck, whose work has been incredibly helpful and inspiring. It has motivated me to learn and improve significantly.

I would like to propose a feature request for a script that simplifies the process of auto-mounting drives on the host. The script would ideally retrieve the necessary details about the drive and automatically add the required lines to the /etc/fstab file. I think this might fit well under the Proxmox VE Tools category.

Unfortunately, I lack the technical expertise to contribute to this feature myself, but I hope this idea can be considered and implemented by the community.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by HearthCore** [comment]
What type of issue are you trying to solve, exactly?
What type of workflow are you currently using to reach your goal?

**Reply by thor9898** [reply]
When I first started using Proxmox, I found it quite challenging to mount my HDD and configure it to auto-mount on reboot. However, afte some practice, I now find it much easier. To help other newcomers, I think a script automating this process would be incredibly useful.

**Comment by HammyHavoc** [comment]
Would also need resiliency so that if for example an SMB mount disconnects, it will check for stale mounts and attempt to reconnect every `x`.
